regles.md

```diff
@@ -0,0 +1,183 @@
+**Not a Rule Book**
+
+**~ ~ ~ Règlement ~ ~ ~**
+
+Nous mettons l'accent sur l'aspect communautaire, il arrive que des nouveaux
+transgressent de manière flagrante les bonnes moeurs du serveur, les quelques
+règles qui suivent sont à leur attention.
+
+1. Le respect des autres membres du serveur est une priorité. Vous soignez vos
+   questions, vos réponses, votre manière de débattre et l'ensemble de vos
+   interactions.
+
+2. La loi française est d'application sur le serveur. Les contenus illégaux, le
+   piratage, la pornographie, le gore, etc. sont interdits. Plus précisément,
+   les sujets borderlines, ceux à la légalité douteuse ou dont l'usage n'est
+   pas clairement dans les clous de la loi sont également prohibés.
+
+3. La publicité, la présentation des projets et les messages de recrutements
+   sont soumis à une règlementation accrue. Il existe des salons dédiés à ces
+   messages et l'accès à ses salons est modéré par le staff. Tout message à
+   tendance publicitaire ou de recrutement envoyé en dehors de ces salons ou
+   par MP est interdit. Voir la section *Les salons modérés*.
+
+4. Les comptes multiples sont interdits, toute tentative pour contourner les
+   sanctions aura pour unique conséquence d'aggraver votre cas.
+
+5. Le serveur promeut l'utilisation d'un français correct et compréhensible.
+   Utiliser un langage dit SMS à excès est interdit. Lorsque vous demandez de
+   l'aide, vous êtes priés de soigner votre message.
+
+6. Le serveur se veut dans l'ensemble sérieux, si il existe des salons prévus
+   aux discussions diverses (#discussions et #detente), c'est pour que les
+   autres salons puissent rester centrés sur leur domaines.
+
+La 1e règle du serveur est essentielle au maintient d'une ambiance propice au
+débat, à l'aide apportée aux nouveaux et plus généralement à un travail serein.
+Le staff se réserve le droit de mettre un terme aux débats foireux, aux dramas
+et aux autres dérives.
+
+**~ ~ ~ Moeurs ~ ~ ~**
```

Comment:
Nous avons choisi de fusionner la FAQ avec le règlement, toutes les sections à partir de celle-ci sont reprises de la FAQ avec quelques modification. Le fait de fusionner règlement et faq est ouvert au débat.

Reply:
Edit, après plus de discussions avec le staff et les ambassadeurs, nous préférons garder la séparation en les règles et la faq. Une référence depuis les règles vers la faq a été ajoutée pour inviter à en prendre connaissance tout en gardant le règlement *to the point*.